notebooks/Stolen_Szechuan_Sauce_Analysis.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/google/timesketch/blob/master/notebooks/Stolen_Szechuan_Sauce_Analysis.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/kiddinn/timesketch/blob/colab_update/notebooks/Stolen_Szechuan_Sauce_Analysis.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
do you wanna reference `kiddinn/timesketch/blob/`here? not master?

Reply:
yes, that's why this is still a draft PR, since I'm not done, I may have to make other changes